connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -92,7 +92,13 @@ private class UCProxy extends TableCatalog {
     this.name
   }
 
-  override def listTables(namespace: Array[String]): Array[Identifier] = ???
+  override def listTables(namespace: Array[String]): Array[Identifier] = {
+    if (namespace.length > 1) {
+      throw new ApiException("Unity Catalog does not support nested namespace.")
+    }
+    val response: ListTablesResponse = tablesApi.listTables(this.name, namespace.head, 0, null)
```

Comment:
What do you think about introducing local variables with names for the input arguments, i.e. `catalogName`, `schemaName`, `maxResults` and `pageToken`? Java doesn't support named arguments unfortunately. I think it would make code comprehension a bit better (unless you're in IntelliJ IDEA and gets all this for free 😉 )

A bit weird to see `null` among the input arguments, isn't it?

Reply:
ah I see. Sure done. This seems to be more readable.